src/extension.ts

```diff
@@ -81,17 +82,33 @@ ${gaugeVersion.stdout.toString()}
     });
 }
 
-function showReferences(languageClient: LanguageClient): any {
-    return (uri: string, position: LSPosition, stepValue: string, count: number) => {
-        if (count > 0) {
-            languageClient.sendRequest("gauge/stepReferences", stepValue, new vscode.CancellationTokenSource().token).then((locations: LSLocation[]) => {
-                vscode.commands.executeCommand('editor.action.showReferences', Uri.parse(uri), languageClient.protocol2CodeConverter.asPosition(position),
-                    locations.map(languageClient.protocol2CodeConverter.asLocation))
-            });
-        }
+function showCodeLensReferences(languageClient: LanguageClient): (uri: string, position: LSPosition, stepValue: string) => Thenable<any> {
```

Comment:
Change name of function. It doesn't depend on CodeLensReferences at all. It is invoked from `showStepReferencesAtCursor` too.